### ISSUE TITLE
Latest update causes issues with multiple redux stores.

### ISSUE THREAD

**Issue by kiwdahc** [zalmoxisus/redux-devtools-extension#610]
In the latest version of Chrome Redux DevTools if you have multiple redux instances which are viewable the inspector always switches to the top instance in the list anytime an action is dispatched from any instance. This is very annoying as the instance I am trying to inspect is the second one in the list and anytime an action is fired the inspector switches to the first instance on the list at which point I must click back to the instance I was previously viewing. 

Our application has multiple apps with redux which are viewable at a time which makes this a very tedious thing to keep clicking back when trying to debug nearly anything.

**Comment by zalmoxisus** [comment]
Thanks for the report. I'll look into it. If you can set a simple example on jsfiddle to reproduce, that would be helpful, as I don't have an example with multiple stores to check atm.

**Comment by zalmoxisus** [comment]
@kiwdahc I modified [our counter example](https://github.com/zalmoxisus/redux-devtools-extension/tree/multiple-stores-example/examples/counter) to [have 2 stores + another 2 in different frames](https://github.com/zalmoxisus/redux-devtools-extension/commit/088f60e18f5af2aeb5103287a3d537dc00cd1e48). As seen bellow it works as expected. For "Autoselect instances" it jumps between instances depending on which dispatched the action. When selecting explecitely an instance, it shows only actions of that instance.

![pdylkryikx](https://user-images.githubusercontent.com/7957859/50176687-bef72380-0308-11e9-9a89-8e01ce02f1f3.gif)

**Comment by kiwdahc** [comment]
I will do some more digging to see what exactly is causing the issue. Will have it early next week along with some images as I am on vacation currently.

**Comment by kiwdahc** [comment]
@zalmoxisus Also another thing of note before I try to recreate the issue besides the above comment is that I never saw this issue until December 13th, our apps were all working properly with the inspector before this. This effect started for me and one of my co-workers at on or before Dec 13th.

**Comment by zalmoxisus** [comment]
@kiwdahc thanks for all the details! Yes, that [autoselecting](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/src/app/middlewares/instanceSelector.js) works only for [devpanel and pageaction popup](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/src/app/stores/windowStore.js#L13), not when opened in a window. And yes it was introduced beginning of December in #580.

I'll do a major refactoring on how the extension communicates in next version and this should be more stable as well.

**Comment by kiwdahc** [comment]
@zalmoxisus It seems one of our applications is stealing focus from any redux instance when any actions are fired in any of the open instances. I haven't found what is different about this instance yet but I am still digging. It even steals focus from a basic jsfiddle redux example. I have noticed though that if the instances are in different browser windows it doesn't steal the instance focus.

**Comment by Empowerful** [comment]
I'm seeing the same behavior.  I have two React applications, one running in the iframe of another, and the instances keep switching automatically on every new action even when I choose one explicitly.

**Comment by kiwdahc** [comment]
Yes this has been happening for us since early December with the same scenario you are having one app in an Iframe with other redux apps inside of it. The only way we are able to currently resolve the issue is either 1. working in an Incognito mode tab, or 2. clicking the external window button in the bottom right hand corner of the redux devtool inspector which opens the inspector into its own window. Not sure why either of these are resolving the issue but both of them prevent the auto-select instance from stealing focus.

**Comment by Empowerful** [comment]
Thank you for sharing your workarounds; I really appreciate it.

**Comment by kiwdahc** [comment]
No problem, this is still an issue for our whole organization and most people using nested redux within iframes. I have just accepted that this problem will probably never be resolved.

**Comment by Draccoz** [comment]
Having the same issue, got 2 angular apps and it keeps switching to the one I am **not** inspecting. Pretty annoying, would love it fixed.

**Comment by Froglegg** [comment]
Using kiwdahc's workaround from Dec 2018 is working for me. I wonder if this issue is caused by using other libraries that have redux as a dependency (e.g., react-beautiful-dnd [as mentioned here](https://github.com/mbrn/material-table/issues/930)).

**Comment by rendomnet** [comment]
same thing here
